2-mandatory/3-financial-times.js

```diff
@@ -23,19 +32,40 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let arr = [];
+
+  for (let moreClick of allArticleTitles) {
+    let choosen = "";
+    for (let letter of moreClick) {
+      if (!isNaN(letter) && letter !== " ") {
+        choosen = moreClick;
+        break;
+      }
+    }
+    if (choosen.length !== 0) {
+      arr.push(choosen);
+    }
+  }
+  return arr;
+
 }
 
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
-    // TODO
+  let total = 0;
+  let average = 0;
+  for (let articleTitle of allArticleTitles) {
+    total += articleTitle.length;
+  }
+  average = total / allArticleTitles.length;
```

Comment:
really nice solution well done, you could just define average on line 63, for example `let average = total / allArticleTitles.length;` and get rid of your definition on line 59 (as it's not really doing anything at the moment)